src/components/BackgroundImage/BackgroundImage.tsx

```diff
@@ -0,0 +1,13 @@
+type BackgroundImageProps = {
+  children: React.ReactNode;
+};
+
+const BackgroundImage = ({ children }: BackgroundImageProps) => {
+  return (
+    <div className="h-[960px] w-full bg-[url('/images/AK_bg_photo1.png')] bg-cover bg-center bg-no-repeat">
```

Comment:
@Okazaki92  I think you need to rework this component. It seems to me that it is advisable here to use image from next.js. This will give us more control in displaying images on the server side (it will improve seo). https://nextjs.org/docs/pages/api-reference/components/image